mono/utils/mono-threads-coop.c

```diff
@@ -510,7 +523,7 @@ mono_threads_are_safepoints_enabled (void)
 gboolean
 mono_threads_is_blocking_transition_enabled (void)
 {
-#if defined(USE_COOP_GC)
+#if defined(USE_HYBRID_COOP)
```

Comment:
We want it for both `USE_COOP_GC` *and* `USE_HYBRID_COOP`.

Reply:
Also, Hybrid Coop doesn't make much sense, Hybrid Suspend would be more meaningful.